### ISSUE TITLE
compare to cligj

### ISSUE THREAD

**Issue by perrygeo** [geojson/py-feature-sequences#5]
This repo seems to overlap with [cligj](https://github.com/mapbox/cligj#example) - we should define the relationship between them.

**Comment by geowurster** [comment]
@perrygeo Yeah - there's some background in https://github.com/geojson/py-feature-sequences/pull/1, but I haven't had time to revisit.  Seems like we want this library to handle getting GeoJSON objects to and from disk and let `cligj` use it for I/O so we can take advantage of any validation.  We would need to support `stdin/stdout` in `geojseq.open()` but that's pretty useful anyway.

`cligj` likes geometries and features, so there are some small design implications in https://github.com/geojson/geojson-feature-sequences/issues/5.
